programs/uxd/src/instructions/credix_lp/edit_credix_lp_depository.rs

```diff
@@ -40,7 +40,7 @@ pub struct EditCredixLpDepository<'info> {
     pub depository: AccountLoader<'info, CredixLpDepository>,
 }
 
-#[derive(AnchorSerialize, AnchorDeserialize, Clone)]
+#[derive(AnchorSerialize, AnchorDeserialize, Clone, Copy)]
```

Comment:
needed to be able to dereference the structure when passing it as a parameter as a reference